services/src/test/scala/cromwell/services/ServicesStoreSpec.scala

```diff
@@ -45,8 +44,14 @@ class ServicesStoreSpec extends FlatSpec with Matchers with ScalaFutures with St
 
         // TODO PBE get rid of this after the migration of #789 has run.
         val oldeTables = Seq(
+          "EXECUTION",
+          "EXECUTION_INFO",
           "EXECUTION_EVENT",
```

Comment:
Don't know. I didn't test removing "EXECUTION_EVENT" from the `oldeTables`. It's possible it may work.